supported/osg-htc/osg-hosted-ce/Chart.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: v1
+appVersion: "5.1.5"
```

```suggestion
appVersion: "V5-branch"
```
updating the appVersion each time we do a new RPM release of htcondor-ce is a pain, and just leaving it at 5.1.5 is misleading. For the osdf-origin chart I used the branch name for the appVersion.

---

Good call, though this is actually a lie anyway. The chart doesn't really enforce which underlying container to use so this could be whatever. But we can provide a reasonable "default" based on what we ship in the latest containers (V6)